src/Element/CustomElement.php

```diff
@@ -297,6 +297,10 @@ private function addFragmentControllerDefaults()
 		$this->Template->element_html_id ??= $this->Template->cssID[0] ?? null;
 		$this->Template->element_css_classes ??= $this->Template->cssID[1] ?? '';
 
+		if (!empty($this->objModel->classes) && \is_array($this->objModel->classes)) {
+			$this->Template->element_css_classes .= ' ' . implode(' ', $this->objModel->classes);
+		}
```

Comment:
```suggestion
		$this->Template->element_css_classes ??= trim(($this->Template->cssID[1] ?? '') . ' ' . implode(' ', $this->objModel ? (array) $this->objModel->classes : []));
```

As we should only set it if `$this->Template->element_css_classes` was not set already (`??=`).